src/OpenSage.Game/Data/W3d/W3dVertexMapperArgs.cs

```diff
@@ -57,6 +57,16 @@ public W3dVertexMapperArgs(W3dChunkType chunkType)
         public float UStep;
         public float VStep;
         public float StepsPerSecond;
+        public float Offset;
+        public float Axis;
+        public float UOffset;
+        public float VOffset;
+        public float ClampFix;
+        public float UseReflect;
+        public float VPreSec;
```

Comment:
Should this be `VPerSec`?

Reply:
Hah, This is not a mistake ... enb assets contain a couple incorrect spellings in the mapper args. Case seems to be different as well. I was quite taken back when I saw this.